src/Orchard.Web/Core/Common/Migrations.cs

```diff
@@ -31,8 +31,8 @@ public class Migrations : DataMigrationImpl {
                     .Column<DateTime>("ModifiedUtc")
                     .Column<int>("Container_id")
                 );
-            
-            SchemaBuilder.CreateTable("CommonPartVersionRecord", 
+
+            SchemaBuilder.CreateTable("CommonPartVersionRecord",
                 table => table
                     .ContentPartVersionRecord()
                     .Column<DateTime>("CreatedUtc")

```

Comment:
It's recommended that you also update this code path so that new installs don't need to run a Create then an Alter table. Then this step will also return 5.